java/ql/src/Telemetry/DatabaseQualityDiagnostics.ql

```diff
@@ -0,0 +1,44 @@
+/**
+ * @name Low Java analysis quality
+ * @description Low Java analysis quality
+ * @kind diagnostic
+ * @id java/diagnostic/database-quality
+ */
+
+import java
+import DatabaseQuality
+
+private newtype TDbQualityDiagnostic =
+  TTheDbQualityDiagnostic() {
+    exists(float percentageGood |
+      CallTargetStatsReport::percentageOfOk(_, percentageGood)
+      or
+      ExprTypeStatsReport::percentageOfOk(_, percentageGood)
+    |
+      percentageGood < 95
+    )
+  }
+
+class DbQualityDiagnostic extends TDbQualityDiagnostic {
+  string toString() {
+    result =
+      "There were significant issues scanning Java code. " +
```

Comment:
> There were significant issues scanning Java code. 

I'm not sure about the usage of `significant`. It feels like a strong argument that this scan is very bad, while it could in fact still be pretty good. 

How does "There were issues scanning Java code."  sound?

Reply:
Amended the message